pages/_app.tsx

```diff
@@ -1,15 +1,37 @@
 // Patternfly import needs to be first or the css will not be imported
 import '@patternfly/react-core/dist/styles/base.css';
 import { Layout } from '@src/components';
-import type { AppProps } from 'next/app';
+import { getAnalytics, getUserRegion } from '@src/util/client';
+import { AppProps } from 'next/app';
+import { useRouter } from 'next/router';
+import { NextPage } from 'next';
+import { useEffect } from 'react';
 
 /**
- * Renders the {@link App}
+ * Renders the {@link MyApp}
  */
-const App: React.FC<AppProps> = ({ Component, pageProps }: AppProps) => (
-  <Layout>
-    <Component {...pageProps} />
-  </Layout>
-);
+const MyApp: NextPage<AppProps> = ({ Component, pageProps }: AppProps) => {
+  const router = useRouter();
 
-export default App;
+  // Client renders page
+  useEffect(() => {
+    const analytics = getAnalytics();
+    const region = getUserRegion(router.locale);
+
+    if (analytics) {
+      analytics.page({
+        userId: 'registry-viewer',
```

Comment:
can we store the client name as a constant somewhere?  It's used in multiple places